types/src/transaction/transaction_v1/transaction_v1_hash.rs

```diff
@@ -23,13 +23,8 @@ use crate::{
     Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Hash, Serialize, Deserialize, Debug, Default,
 )]
 #[cfg_attr(feature = "datasize", derive(DataSize))]
-#[cfg_attr(
-    feature = "json-schema",
-    derive(JsonSchema),
-    schemars(with = "String", description = "Hex-encoded TransactionV1 hash.")
-)]
 #[serde(deny_unknown_fields)]
-pub struct TransactionV1Hash(#[cfg_attr(feature = "json-schema", schemars(skip))] Digest);
+pub struct TransactionV1Hash(Digest);
```

Comment:
This is a good change, but we should probably apply it similarly to `BlockHash` and `DeployHash`.